src/main/java/net/mcreator/blockly/BlocklyBlockUtil.java

```diff
@@ -118,4 +119,49 @@ public static Color getBlockColorFromHUE(int hue) {
 		return Color.getHSBColor(hue / 360f, 0.37f, 0.6f);
 	}
 
+	/**
+	 * Calculate the HUE of a block from RGB
+	 *
+	 * @param var The variable element to use
+	 * @return Built-in color or HUE for other blocks
+	 */
+	public static String getHUEFromRGB(VariableElementType var) {

```

Comment:
var is reserved Java world in newer Java versions. Rename all these variables accross this PR or MCreator will break when we update Java.

Reply:
Oh, I didn't know. I never used another version other than Java 8.

Reply:
I plan to switch to J9+ if possible in the next release so it may/will matter in the future